app/renderer/js/utils/config-util.js

```diff
@@ -49,6 +49,16 @@ class ConfigUtil {
 		}
 	}
 
+	isConfigItemExists(key) {
+		this.reloadDB();
+		const value = this.db.getData('/')[key];
+		if (value === undefined) {
+			return false;
+		} else {
+			return true;
+		}
+	}
+
```

Comment:
This can be also be done like -
```js
	isConfigItemExists(key) {
		this.reloadDB();
		const value = this.db.getData('/')[key];
		return (value === undefined);
	};
```

Reply:
Shouldn't it be 
```
return !(value === undefined);
```